recipes/configs/dev/8B_full_experimental.yaml

```diff
@@ -18,6 +18,8 @@
 # best to use 8B_full_single_device.yaml for those cases
 
 
+output_dir: /tmp/torchtune/dev_8B/full_experimental # /tmp may be deleted by your system. Change it to your preference.
```

Comment:
What is this recipe? Shouldn't it be under a dev/model structure?

Reply:
Created #2114 to continue the discussion there